x-pack/legacy/plugins/epm/public/screens/add_data_source/add_data_source_steps.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useState, Fragment, useCallback } from 'react';
+import styled from 'styled-components';
+import { EuiPanel, EuiSteps, EuiButton, EuiHorizontalRule } from '@elastic/eui';
+import { Redirect } from 'react-router-dom';
+import { StepOneTemplate } from './step_one';
+import { installDatasource } from '../../data';
+import { useCore, useLinks } from '../../hooks';
+
+const StyledSteps = styled.div`
+  .euiStep__titleWrapper {
```

Comment:
Ideally we wouldn't target these internal selectors but if that's the only way 🤷‍♀

Reply:
yea, unfortunately EuiSteps only allows a string into the title or I could pass an element to style.  Henry recommended I do something like this to have a custom styled title. I'll make a note to mention it to the eui team about making it more flexible with an element.